src/rust/inetstack/protocols/mod.rs

```diff
@@ -10,12 +14,32 @@ pub mod peer;
 pub mod tcp;
 pub mod udp;
 
+//======================================================================================================================
+// Imports
+//======================================================================================================================
+
 use ::std::slice::ChunksExact;
 
+//======================================================================================================================
+// Constants
+//======================================================================================================================
+
+pub const MAX_HEADER_SIZE: usize =
```

Comment:
I understand that this should be defined as the MAXIMUM header among all the protocols that are supported. Could you add a comment about that?